integrations/qdrant/src/haystack_integrations/document_stores/qdrant/document_store.py

```diff
@@ -187,6 +320,15 @@ def filter_documents(
         self,
         filters: Optional[Union[Dict[str, Any], rest.Filter]] = None,
     ) -> List[Document]:
+        """
+        Returns the documents that match the provided filters.
+
+        For a detailed specification of the filters, refer to the
+        DocumentStore.filter_documents() protocol documentation.
```

Comment:
```suggestion
        For a detailed specification of the filters, refer to the
        [documentation](https://docs.haystack.deepset.ai/docs/metadata-filtering).
```

This seems more user-friendly.